tests/integration/google_cloud_storage/test_google_cloud_storage.py

```diff
@@ -60,34 +58,39 @@ def google_cloud_storage_configured():
 )
 @pytest.mark.parametrize("use_datastore_profile", [False, True])
 class TestGoogleCloudStorage:
-    #  must be static, in order to get access from setup_class or teardown_class.
-    @staticmethod
-    def clean_test_directory(bucket_name, object_dir, gcs_fs):
-        test_dir = f"{bucket_name}/{object_dir}/"
-        if gcs_fs.exists(test_dir):
-            gcs_fs.delete(test_dir, recursive=True)
+    assets_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), "assets")
+    _bucket_name = config["env"].get("bucket_name")
```

Comment:
```suggestion
    bucket_name = config["env"].get("bucket_name")
```
Unless I'm missing something, there's no reason for this attribute in particular to be private.

Reply:
I meant everywhere though, including `TestAlibabaOssDataStore`.